CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ## UNRELEASED
 
+### Added
```

Comment:
"Added" seems to be quite coiunter-intuitive if something has been removed.

Reply:
or at least changed